src/components/dashboard/FaceRecognition/ZoomCapture.js

```diff
@@ -108,7 +99,7 @@ const HelperWizard = props => {
     <React.Fragment>
       <View id="background" style={styles.background} />
       <View style={{ zIndex: 10, justifyContent: 'space-evenly', height: '100%' }}>
-        <Text style={{ fontFamily: 'Roboto-Medium', fontSize: normalize(20), color: 'white' }}>{text}</Text>
+        <Text style={{ fontFamily: 'Roboto-Medium', fontSize: 20, color: 'white' }}>{text}</Text>
```

Comment:
The Text component already receive those styles as props, so you can use 
```suggestion
    <Text fontFamily="medium" fontSize={20} color={surface}>{text}</Text>
```

Reply:
I don't think I've worked on this.